docs/central-upgrade.rst

```diff
@@ -76,14 +77,170 @@ You'll be asked to confirm the removal of all dangling images. Agree by typing t
+
+    #. Log into your server. If you used our :doc:`DigitalOcean installation steps <central-install-digital-ocean>`, we suggest reviewing the section :ref:`central-install-digital-ocean-build` as a reminder, or if you can't remember your password to start at the top of that section to reset your password.
+
+    #. Get the latest infrastructure version.
+
+       .. code-block:: bash
+
+          cd central
+          git pull
+
+       .. note::
+
+          If you have made local changes to the files, you may have to start with ``git stash``, then run ``git stash pop`` after you perform the ``pull``. If you aren't sure, run ``git pull`` and it will tell you.
+
+    #. Get the latest client and server.
+
+       .. code-block:: bash
+
+          git submodule update -i
+
+    #. Check that you have enough disk space available. If you are prompted for a password, enter the system superuser password (not a Central password).
+
+       .. code-block:: bash
+
+          sudo ./files/postgres14/upgrade/check-available-space
+
+       If you don't have enough space, stop here and resume when you have increased the disk space available. You may achieve this by clearing out data you don't need (e.g., logs) or by increasing the total disk space available (e.g., by :ref:`adding external storage <central-install-digital-ocean-external-storage>`).
+
+    #. Create a file to prove that you're carefully reading these instructions. This is required to continue.
+
+       .. code-block:: bash
+
+          touch ./files/allow-postgres14-upgrade
+
+    #. Stop Central.
+
+       .. code-block:: bash
+
+          docker-compose stop
+
+    #. Build from the latest code you just fetched.
+
+       .. code-block:: bash
+
+          docker-compose build
+
+    #. Start the database upgrade and wait for the process to exit.
+
+       .. code-block:: bash
+
+          docker-compose up postgres
+
+    #. Check the output of the previous command to see if there were any errors. If there were any errors that you can't resolve, `write a support post on the forum <https://forum.getodk.org/c/support/6>`_.
+
+    #. Check the upgrade success file has been created.
+
+       .. code-block:: bash
+
+          ls ./files/postgres14/upgrade/upgrade-successful
+
+       If you see "No such file or directory," try doing ``docker-compose up postgres`` again. If the file has still not been created, `write a support post on the forum <https://forum.getodk.org/c/support/6>`_.
+
+    #. Restart the server and verify that everything works as expected.
+
+       .. code-block:: bash
+
+            docker-compose up -d
+
+    #. Log into the web interface and do some quick spot checks. For example, verify that submission counts and latest submission dates look right and try a data export.
+
+    #. The upgrade process performs a copy and leaves the old database intact. The following command will show you how much space this old database takes.
```

Comment:
Here, the steps switch from "You're done! You did it! Hopefully eerything looks good!" to more... optional/less time critical cleanup. I think this should be a separate section so you can know when you're done with the hardest part.